daemon/graphdriver/aufs/aufs_test.go

```diff
@@ -179,9 +181,8 @@ func TestCleanupWithNoDirs(t *testing.T) {
 	d := newDriver(t)
 	defer os.RemoveAll(tmp)
 
-	if err := d.Cleanup(); err != nil {
-		t.Fatal(err)
-	}
+	err := d.Cleanup()
+	require.NoError(t, err)
```

Comment:
Could possibly even be;

```go
require.NoError(t, d.Cleanup())
```

Reply:
Could be, but I try to keep the function being tested on its own line. I think the test reads better when the function under test is separate from the assertions.